cmd/kubeadm/app/phases/addons/proxy/manifests.go

```diff
@@ -175,6 +175,20 @@ spec:
         - mountPath: /run/xtables.lock
           name: xtables-lock
           readOnly: false
+	  initContainers:
+      - name: init-kube-proxy
+        image: gcr.io/google_containers/busybox
+        command:
+        - /bin/sh
+        - -c
+        - cat /var/lib/kube-proxy/config.conf | grep "mode: ipvs" &&
```

Comment:
Instead of parsing the file on disk, can we pull in the [component configmap](url) ?

Reply:
using `configmap` to config conf will be better

Reply:
Thanks for reviewing. I tried to use configmap. Kubeadm created kube-proxy configmap with a struct:
```yaml
apiVersion: v1
data:
  config.conf: |-
    apiVersion: kubeproxy.config.k8s.io/v1alpha1
...
    mode: ipvs
...
```
In which `config.conf` is a key with a bug value. I can use this key get all `config.conf` value and see it include `ipvs: mode` or not. But can't get `mode` field. Is there any other way of using configmap?